sovtoken/sovtoken/request_handlers/batch_req_handler/token_batch_handler.py

```diff
@@ -24,13 +25,4 @@ def post_batch_applied(self, three_pc_batch, prev_handler_result=None):
 
     def commit_batch(self, three_pc_batch, prev_handler_result=None):
         super().commit_batch(three_pc_batch, prev_handler_result)
-        state_root = base58.b58decode(three_pc_batch.state_root.encode()) if isinstance(
-            three_pc_batch.state_root, str) else three_pc_batch.state_root
-        if self.utxo_cache.first_batch_idr != state_root:
-            raise TokenValueError(
-                'state_root', state_root,
-                ("equal to utxo_cache.first_batch_idr hash {}"
-                 .format(self.utxo_cache.first_batch_idr))
-            )
-        self.utxo_cache.commit_batch()
-
+        commit_to_utxo_cache(self.utxo_cache, three_pc_batch.state_root)
```

Comment:
I think, that we should move utxo_cache's operations into UTXOCacheBatchHandler (similar with idr_cache)

Reply:
ok